hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -560,6 +560,94 @@
     }
   ]
 
+  features: [
```

Comment:
Please add features for integrity and consistency checks. They have period, timeout, and direct checks subfeatures.

Reply:
These are dai micro operations I mentioned [here](https://github.com/lowRISC/opentitan/pull/19889#discussion_r1373807186).
Feature is added but not won't be able to check in sival.